utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -0,0 +1,414 @@
+#include <stdio.h>
```

Comment:
I see brightness_u8_Tensor_PKD3.csv file name is good. Need to change others too.

Reply:
Done